frontend/src/index.css

```diff
@@ -0,0 +1,5 @@
+body,
+html {
+  margin: 0 !important;
```

Comment:
Maybe you could use Globalstyles.js?

Reply:
Would also recommend that, !important is not the best way.